File: python/atm_pin.py
#chance,correct pin,user pin
user = int(input("Enter pin: "))
correct_pin = 4356
chance = 1
while True:    
    if user==correct_pin:
        print("Access Granted")
        break
    if chance==3:
        print("Card Blocked")
        break
    else:
        print("Incorrect")
        user = int(input("Enter pin: "))
        chance+=1

File: C/atm_pin.c
//This is the 2st question from 5th ILBP practical

#include <stdio.h>

int main(void){
	int attempt=1,pin=4356;
	while (1){
		int user;
		printf("\nEnter your ATM pin: ");
		scanf("%d",&user);
		if (attempt == 3){
			printf("\nCard Blocked \n");
			break;
		}
		if (user == pin){
			printf("\nAccess Granted\n");
			break;
		}
		else{
			printf("\nIncorrect try again");
			attempt+=1;
		}
	}
}
